starter-code/src/functions-and-arrays.js

```diff
@@ -10,15 +14,41 @@ var words = [
   'orchard',
   'crackpot'
 ];
-
+function findLongestWord(words){
+  let index= 0;
+  let max = words[index];
+  for(var a =0; a<words.length;a++){
+    if(words[a].length > max.length) {
+     max = words[a];
+    }
+  }
+  return max;
+}
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
-
+function sumArray(numbers){
+  let sum=0;
+  for(var a =0;a<numbers.length;a++){
+    sum += numbers[a];
+  }
+  return sum;
+}
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(numbersAv){
```

Comment:
In this function sometimes you call numbersAvg 'numbersAv', and it crashes the application. The function works, otherwise.